src/finch/julia.py

```diff
@@ -1,6 +1,6 @@
 import juliapkg
 
-_FINCH_VERSION = "0.6.23"
+_FINCH_VERSION = "0.6.24"
 _FINCH_HASH = "9177782c-1635-4eb9-9bfb-d9dfa25e6bce"
```

Comment:
Does this hash need updating?

Reply:
No, hash stays the same, it's associated with the package.